packages/subproviders/src/subproviders/injected_web3.ts

```diff
@@ -43,7 +42,7 @@ export class InjectedWeb3Subprovider {
     }
     // Required to implement this method despite not needing it for this subprovider
     // tslint:disable-next-line:prefer-function-over-method
-    public setEngine(engine: Web3ProviderEngine) {
+    public setEngine(engine: any) {
```

Comment:
Why was this changed?

Reply:
Since we're not using the type in any way its only use was to be imported and then referenced here. By removing the type and using any in a method that is a noop, we can remove the import of the Web3 Provider Engine in this class.